webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/properties/MergeNodeProperties.scala

```diff
@@ -289,16 +286,25 @@ object MergeNodeProperties {
           }
       }
     }
+    def resolveHierarchy(group: FullGroupTarget): ResolvedNodePropertyHierarchy = {
+      val resolved = for {
+        hierarchy   <-
+          // this fails with an empty list of property, maybe we should also ignore group parent errors, as with "sortGroups"
+          Ior.fromEither(withParents(group :: Nil, allGroups.map { case (id, group) => (id.serialize, group) }, Map()))
```

Comment:
I think here too there is the same compromise : should we fail the resolution of properties completely, or do the "best effort" ?